src/iface/interface/mod.rs

```diff
@@ -398,10 +398,11 @@ impl Interface {
     /// have changed.
     ///
     /// # Note
-    /// If this function returns true, there may still be packets to be transmitted or received.
-    /// Users should should perform other tasks before calling this function again. If executed in
-    /// a loop, its possible that packets can arrive faster than they can be processed, which could
-    /// cause an infinite loop.
+    /// This function performs a bounded amount of work per call to avoid exhausting
```

Comment:
Oh, typo:

```suggestion
    /// This function performs a bounded amount of work per call to avoid
```

(I guess this also makes formatting uglier... sigh)

Reply:
Yeah sorry, I saw this already and removed it, and indeed the formatting is ugly